draft-yasskin-http-origin-signed-responses.md

```diff
@@ -838,6 +838,10 @@ signature returns "valid", return "valid". Otherwise, return "invalid".
 1. Let `exchange` be the exchange metadata and headers parsed out of `headers`.
 1. If `exchange`'s request method is not safe (Section 4.2.1 of {{!RFC7231}}) or
    not cacheable (Section 4.2.3 of {{!RFC7231}}), return "invalid".
```

Comment:
Let's replace the "or not cacheable" bit with the reference to https://httpwg.org/specs/rfc7234.html#rfc.section.3, since that also talks about methods.

Reply:
Done